diff --git a/src/applications/facility-locator/api/index.js b/src/applications/facility-locator/api/index.js
index fb2047094..63825af60 100644
--- a/src/applications/facility-locator/api/index.js
+++ b/src/applications/facility-locator/api/index.js
@@ -2,4 +2,4 @@ import MockApi from './MockLocatorApi';
 import LiveApi from './LocatorApi';
 import environment from '../../../platform/utilities/environment';
 
-export default (environment.isLocalhost() ? MockApi : LiveApi);
+export default (environment.isLocalhost() ? LiveApi : LiveApi);
diff --git a/src/applications/facility-locator/config.js b/src/applications/facility-locator/config.js
index f10a72e66..cf930f1f1 100644
--- a/src/applications/facility-locator/config.js
+++ b/src/applications/facility-locator/config.js
@@ -1,10 +1,10 @@
-import environment from '../../platform/utilities/environment';
+//import environment from '../../platform/utilities/environment';
 import { LocationType, FacilityType } from './constants';
 
-// TODO: Remove me when done bug fixing
-// const environment = {
-//   API_URL: 'http://staging-api.va.gov',
-// };
+ //TODO: Remove me when done bug fixing
+ const environment = {
+   API_URL: 'http://localhost:3000',
+ };
 
 // Base URL to be used in API requests.
 export const api = {
