diff --git a/src/applications/facility-locator/components/markers/DivMarker.jsx b/src/applications/facility-locator/components/markers/DivMarker.jsx
index baa8df718..607b5705c 100644
--- a/src/applications/facility-locator/components/markers/DivMarker.jsx
+++ b/src/applications/facility-locator/components/markers/DivMarker.jsx
@@ -12,11 +12,7 @@ export default class DivMarker extends Component {
       html: renderToStaticMarkup(children),
     });
 
-    return (
-      <Marker icon={icon} position={position} onClick={onClick}>
-        <Popup>{popupContent}</Popup>
-      </Marker>
-    );
+    return <Marker icon={icon} position={position} onClick={onClick} />;
   }
 }
 
