diff --git a/src/applications/facility-locator/components/SearchControls.jsx b/src/applications/facility-locator/components/SearchControls.jsx
index ac84b0c83..5775fb9ad 100644
--- a/src/applications/facility-locator/components/SearchControls.jsx
+++ b/src/applications/facility-locator/components/SearchControls.jsx
@@ -119,38 +119,45 @@ class SearchControls extends Component {
       <div className="search-controls-container clearfix">
         <form
           id="facility-search-controls"
-          className="row"
           onSubmit={this.handleSubmit}
         >
-          <div className="columns medium-3-5">
-            <label
-              htmlFor="street-city-state-zip"
-              id="street-city-state-zip-label"
-            >
-              Search city, state, or postal code
-            </label>
-            <input
-              id="street-city-state-zip"
-              name="street-city-state-zip"
-              type="text"
-              onChange={this.handleQueryChange}
-              value={currentQuery.searchString}
-              title="Your location: Street, City, State or Zip"
-              required
-            />
-          </div>
-          <div className="columns medium-3-5">
-            <FacilityTypeDropdown
-              facilityType={this.props.currentQuery.facilityType}
-              onChange={this.handleFacilityTypeChange}
-              showCommunityCares={showCommunityCares}
-            />
-          </div>
-          <div className="columns medium-3-4">
-            {this.renderServiceTypeDropdown()}
-          </div>
-          <div className="columns medium-1-2">
-            <input type="submit" value="Search" />
+          <div className="row">
+            <div className="columns marg-left">
+              <div className="row">
+                <div className="columns large-1-2">
+                  <label
+                    htmlFor="street-city-state-zip"
+                    id="street-city-state-zip-label"
+                  >
+                    Search city, state, or postal code
+                  </label>
+                  <input
+                    id="street-city-state-zip"
+                    name="street-city-state-zip"
+                    type="text"
+                    onChange={this.handleQueryChange}
+                    value={currentQuery.searchString}
+                    title="Your location: Street, City, State or Zip"
+                    required
+                  />
+                </div>
+              </div>
+              <div className="row">
+                <div className="columns large-1-2">
+                  <FacilityTypeDropdown
+                    facilityType={this.props.currentQuery.facilityType}
+                    onChange={this.handleFacilityTypeChange}
+                    showCommunityCares={showCommunityCares}
+                  />
+                </div>
+                <div className="columns large-1-2">
+                  {this.renderServiceTypeDropdown()}
+                </div>
+                <div className="columns medium-1-2">
+                  <input type="submit" value="Search" />
+                </div>
+              </div>
+            </div>
           </div>
         </form>
       </div>
diff --git a/src/applications/facility-locator/sass/facility-locator.scss b/src/applications/facility-locator/sass/facility-locator.scss
index 531d02224..ed87b02a2 100644
--- a/src/applications/facility-locator/sass/facility-locator.scss
+++ b/src/applications/facility-locator/sass/facility-locator.scss
@@ -339,6 +339,9 @@ $color-info-bubble-text: #818a8d;
         .medium-3-5 {
           width: 31% !important;
         }
+        .large-1-2 {
+          width: 396px !important;
+        }
       }
 
       select:disabled {
@@ -728,6 +731,10 @@ $color-info-bubble-text: #818a8d;
   margin-right: 0 !important;
 }
 
+.marg-left {
+  margin-left: 26px !important;
+}
+
 .pull-right {
   float: right;
 }
